server/src/controllers/Events/resolver.ts

```diff
@@ -289,38 +286,6 @@ const getNameForNewRsvp = (event: EventRsvpName) => {
 
 @Resolver()
 export class EventResolver {
-  @Query(() => [EventWithRelations])
-  async events(
-    @Arg('limit', () => Int, { nullable: true }) limit?: number,
-    @Arg('showAll', { nullable: true }) showAll?: boolean,
-  ): Promise<EventWithRelations[]> {
```

Comment:
I can't believe this has hung around so long.  Thanks for the clean up!

Reply:
It might have been still used somewhat recently in events dashboard, before `dashboardEvents` query was made.

Reply:
Most likely. I probably just forgot that /events doesn't use it 🤷